src/drivers/gitlab.js

```diff
@@ -162,7 +162,7 @@ class Gitlab {
         --token "${token}" \
         --wait-timeout ${idleTimeout} \
         --executor "${IN_DOCKER ? 'shell' : 'docker'}" \
-        --docker-image "dvcorg/cml:latest" \
+        --docker-image "iterativeai/cml:latest" \
```

Comment:
do we have @dmpetrov approval for this? originally it was setup under the umbrella of dvcorg on purpose